pyproject.toml

```diff
@@ -0,0 +1,199 @@
+# https://packaging.python.org/en/latest/specifications/pyproject-toml/
+[build-system]
+requires = ["pdm-backend", "wheel>=0.42"]
+build-backend = "pdm.backend"
+
+[project]
+name = "pymediainfo"
+description = "A Python wrapper for the mediainfo library."
+authors = [
+    {name = "Louis Sautier", email = "sautier.louis@gmail.com"},
+]
+readme = "README.rst"
+license = {text = "MIT"}
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: Implementation :: PyPy",
+]
+dynamic = ["version"]
+requires-python = ">=3.9"
+dependencies = [
+]
+
+[project.optional-dependencies]
+tests = [
+    "pytest>=6",
+    "pytest-cov",
+    "pytest-xdist",
+    "coverage[toml]>=7.2",
+]
+docs = [
+    "alabaster",
+    "setuptools_scm",
+    "sphinx",
+]
+dev = [
+    "ipython",
+    "mypy>=1.0",
+    "black",
+    "isort",
+    "flake8",
+    "pylint",
+]
+
+[project.urls]
+Homepage = "https://github.com/sbraz/pymediainfo"
+Documentation = "https://pymediainfo.readthedocs.io/"
+Bugs = "https://github.com/sbraz/pymediainfo/issues"
+
+
+# https://pdm-project.org/latest/
+[tool.pdm.version]
+source = "scm"
```

Comment:
Does it mean `setuptools_scm` is now only used when building the documentation? Should it be removed from tox.ini as well (except for the doc)?

Reply:
From the `[testenv]` section? Yes, I missed it

Reply:
`pdm` supports get tag versioning natively, that's cool.

Reply:
I wonder if there would be a better way than the ugly line I'm currently using in `docs/conf.py`:
```
version = setuptools_scm.get_version(root="..", fallback_root="..", relative_to=__file__)
```
Let's keep `setuptools_scm` for now. I'll look into this later.

Reply:
What I have seen in other projects is to import the package and use `package.__version__`

Reply:
But it can go in another PR

Reply:
> What I have seen in other projects is to import the package and use `package.__version__`

I wonder if it worked with readthedocs but then again readthedocs changed a lot since I started using it. Maybe they install the package before building the doc nowadays, I don't remember.

> But it can go in another PR

Works for me.